core/common/src/rpc/core/RpcControl.ts

```diff
@@ -105,9 +105,7 @@ export class RpcControlChannel {
     this._configuration.interfaces().forEach((definition) => interfaces.push(`${definition.interfaceName}@${definition.interfaceVersion}`));
     const id = interfaces.sort().join(",");
 
-    if (typeof (btoa) !== "undefined")
-      return btoa(id);
-    else if (typeof (Buffer) !== "undefined")
+    if (typeof (Buffer) !== "undefined")
```

Comment:
@swbsi does this change look okay to you? I assume we were checking for btoa so that we supported older Node versions.